app/css/app.css

```diff
@@ -343,3 +343,16 @@ th.jh-key{
     position: absolute;
     z-index: 3;
 }
+img{
+    height: 80px;
+}
```

Comment:
Yes. But i can also change it to .mug. For now there seems only mug on the page

Sent from my iPhone

> On 29 Jul 2015, at 16:29, Hervé BREDIN <notifications@github.com> wrote:
> 
> In app/css/app.css:
> 
> > @@ -343,3 +343,16 @@ th.jh-key{
> >      position: absolute;
> >      z-index: 3;
> >  }
> > +img{
> > +    height: 80px;
> > +}
> Are you sure this is what you want to do?
> 
> —
> Reply to this email directly or view it on GitHub.
>